inference-engine/src/mkldnn_plugin/nodes/mkldnn_memory_node.hpp

```diff
@@ -89,6 +89,9 @@ class MKLDNNMemoryInputNode : public MKLDNNInputNode, public MKLDNNMemoryNode {
     bool created() const override {
         return getType() == MemoryInput;
     }
+    bool isExecutable() const override {
+        return true;
+    }
```

Comment:
Node `MKLDNNMemoryInput` is child of `MKLDNNInput`. `MKLDNNInput` has this overrided method from base class and it returns `false`. 

So `MKLDNNMemoryInput` has this method with `false` from `MKLDNNInput` (because it's child) but it's incorrect because `MKLDNNMemoryInput`  is executable node. So I should override this method one more time